src/workers/worker.rs

```diff
@@ -200,25 +196,30 @@ impl Worker {
 
         match req.req.clone() {
             ValidateRequest::AdmissionRequest(adm_req) => {
-                if let Some(namespace) = &pes.always_accept_admission_reviews_on_namespace {
-                    // If the policy server is configured to
-                    // always accept admission reviews on a
-                    // given namespace, just set the `allowed`
-                    // part of the response to `true` if the
-                    // request matches this namespace. Keep
-                    // the rest of the behaviors unchanged,
-                    // such as checking if the policy is
-                    // allowed to mutate.
-
-                    if adm_req.namespace == Some(namespace.to_string()) {
+                // TODO: we should check immediately if the request is coming from the "always
```

Comment:
we should track this as an issue / tech-debt

Reply:
Done with https://github.com/kubewarden/policy-server/issues/599